<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(T)</string>
    </property>
    <addaction name="action_mi"/>
    <addaction name="action_jia"/>
    <addaction name="action_jie"/>
   </widget>
   <addaction name="menu_T"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_mi">
   <property name="text">
    <string>生成密钥</string>
   </property>
  </action>
  <action name="action_jia">
   <property name="text">
    <string>加密</string>
   </property>
  </action>
  <action name="action_jie">
   <property name="text">
    <string>解密</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
